x/leverage/simulation/operations.go

```diff
@@ -175,12 +175,16 @@ func SimulateMsgBorrowAsset(ak simulation.AccountKeeper, bk types.BankKeeper, lk
 		r *rand.Rand, app *baseapp.BaseApp, ctx sdk.Context,
 		accs []simtypes.Account, chainID string,
 	) (simtypes.OperationMsg, []simtypes.FutureOperation, error) {
-		from, coin, skip := randomCollateralFields(r, ctx, accs, lk)
+		from, uToken, skip := randomCollateralFields(r, ctx, accs, lk)
```

Comment:
When borrowing assets, a user can borrow any accepted denom, not just those whose uTokens they have in their collateral.

This function should therefore select a random `Token.BaseDenom` from `lk.GetAllRegisteredTokens` and then generate a random amount.

Reply:
done at https://github.com/umee-network/umee/pull/433/commits/6f97a9268077f31aed9f5185b9287a178c935ffd